tests/integration/s2n_handshake_test_gnutls-cli.py

```diff
@@ -323,6 +327,25 @@ def main():
                         " Priority string: " + priority_str)
                 return -1
 
+    # Produce permutations of every curve s2n supports in every possible order
+    curves = ["CURVE-SECP256R1", "CURVE-SECP384R1", "CURVE-SECP521R1"]
```

Comment:
GnuTLS and s2n support x25519, why aren't we including(CURVE-X25519) here for testing?

Reply:
yes, will add it.